src/components/Intranet/Products/ProductInfoModal.tsx

```
@@ -0,0 +1,139 @@
+        if (productId) {
+            (async () => {
+                await fetchProduct();
+            })();
+        }
+    }, [productId, fetchProduct]);
+
+    return (
+        <Modal isOpen={effectiveIsOpen} onOpenChange={handleModalOpenChange}>
+            <ModalContent>
+                <ModalBody className="py-6">
+                    {product && product.image_url ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)